.gitignore

```diff
@@ -17,6 +17,7 @@ __pycache__/
 *$py.class
```

Comment:
Please add a *.json to avoid uploading service account json files

Reply:
Adding `*.json` would break many packages, namely JavaScript/Node.js, where `package.json`, `package-lock.json`, etc. are required.

Additionally, I'm not seeing a convention for service account file names, thus it would be difficult to determine whether a file is actually a service account file.

Reply:
Yeah but you are not using Node.js in this project ? or is the PR missing some files ?

Reply:
Not in this project, but there's about 504 existing JSON files in the repo utilized by JavaScript, Node.js, etc.

Additionally, as mentioned, I don't think it solves the root problem where the file may not be saved with `.json`.

Reply:
Ahh i see what's happening here, you should not edit the .gitignore file in the root of the repo. Please add a .gitignore under tools/sts-job-manager and add the files you want to ignore there.

Reply:
Added & migrated

Reply:
FWIW you can use pre-commit hooks to detect private keys rather than relying on filenaming convention and gitignore (note that this does not protect against someone renaming the key e.g. `mv my-key.json my-key.txt`).
All gcp SA keys have a private key in them ([docs](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys)),
In other projects I use pre-commit to detect-private-keys https://pre-commit.com/hooks.html.
Perhaps it would be good for us to introduce a pre-commit in this repo though it may be difficult to make sure everything is compliant with all the checks we would want to add.

Reply:
Thanks @jaketf. I will look into adding this to the root repo